packages/core/src/Connect.ts

```diff
@@ -0,0 +1,49 @@
+import ConnectorEthereum, {
+  ConnectorEthereumConfig,
+} from 'plumbery-connector-ethereum'
+import ConnectorTheGraph, {
+  ConnectorTheGraphConfig,
+} from 'plumbery-connector-thegraph'
+import ConnectorJson, { ConnectorJsonConfig } from './ConnectorJson'
+import Organization from './wrappers/Organization'
+import { ConnectorInterface } from './ConnectorTypes'
+
+type ConnectorDeclaration = ConnectorInterface | [string, object | undefined]
+
+function getConnector(connector: ConnectorDeclaration): ConnectorInterface {
+  if (!Array.isArray(connector)) {
+    return connector
+  }
+
+  const [name, config = {}] = connector
+
+  if (name === 'json') {
+    return new ConnectorJson(config as ConnectorJsonConfig)
+  }
+  if (name === 'thegraph') {
+    return new ConnectorTheGraph(config as ConnectorTheGraphConfig)
+  }
+  if (name === 'ethereum') {
+    return new ConnectorEthereum(config as ConnectorEthereumConfig)
+  }
+
+  throw new Error(`Unsupported connector name: ${name}`)
+}
+
+type ResolveIpfs = (ipfsIdentifier: string, path: string) => string
+type ResolveOrganization = (location: string) => Organization
+
+export function Connect(
+  { connector, ipfs }: { connector: ConnectorDeclaration; ipfs: ResolveIpfs },
+  location?: string
+): Organization | ResolveOrganization {
+  // TODO: Handle ENS names
```

Comment:
Do you think it makes sense to handle ENS at this level and always initialize the organizations with the address?

Reply:
Why do you consider we should support a configurable ENS resolver? For those cases, users want a custom resolver?

Not sure how much value that would add 🤔

Reply:
I was thinking that we might want to resolve these from a different source than the rest, the same way we might have different sources for the organizations and the apps. What do you think?

Reply:
Yes, make sense, I think we can have an optional option that is the ens registry to be used.